common/djangoapps/cme_registration/models.py

```diff
@@ -103,14 +103,13 @@ class Meta:
     sub_specialty = models.CharField(blank=True, null=True, max_length=255)
 
     AFFILIATION_CHOICES = (('Stanford Children\'s Health', 'Stanford Children\'s Health'),
-                           ('Packard Children\'s Health Alliance', 'Packard Children\'s Health Alliance'),
+                           ('Packard Children\'s Health Alliance (PCHA)', 'Packard Children\'s Health Alliance (PCHA)'),
                            ('Stanford Health Care', 'Stanford Health Care'),
                            ('Stanford University', 'Stanford University'),
-                           ('University Healthcare Alliance', 'University Healthcare Alliance'),
-                           ('Other', 'Other'),
+                           ('University Healthcare Alliance (UHA)', 'University Healthcare Alliance (UHA)'),
+                           ('Not affiliated with Stanford Medicine', 'Not affiliated with Stanford Medicine'),
                            )
     affiliation = models.CharField(blank=True, null=True, max_length=46, choices=AFFILIATION_CHOICES)
-    other_affiliation = models.CharField(blank=True, null=True, max_length=46)
```

Comment:
No need to create another PR. The workflow we follow is to just make changes on the existing PR. Generally add another commit, or you can amend the last commit, and the push back up to github.

If you end up with several commits then generally you'll need to squash them before merging to master.